### PR TITLE
test: add some example in chunk_test.go and ioutil_test.go

### LINKED ISSUE #257: Request help: Supplementary of Unit Tests
> We hope more developers join CloudWeGo. This is a task issue that needs your help.

**Issue Description**
Write Unit Tests for some modules of Hertz, we mainly focus on **Statement Coverage**. The packages are shown below, you can choose and reply by comment.
Before you pick the task, please comply with the following requirements:
- The Unit Test you add must be meaningful. 
  - It is used to verify the correctness of code logic.
  - It is forbidden to add a test only for the purpose of coverage but without logical meaning.
- It is suggested to add comments with clear descriptions for each unit test.
- Every unit test must have assertions.  
- Don't check code logic by `print`.

**Packages**
> Current Coverages v0.3.1.
---
- [ ] `pkg/app`: Current  70.87%,  Target **80%**, by @chaoranz758 
- [ ] `pkg/route`: Current 78.49%,  Target **85%**, by 
- [x] `pkg/protocol/http1`: Current 33.94%,  Target **80%**, by @a631807682 
- [x] `pkg/protocol/uri.go`: Current 70.35%,  Target **80%**, Raised to 86.4%, by @L2ncE
- [x] `pkg/protocol/args.go`: Current 56.57%,  Target **80%**, Raised to 81.3%, by @justlorain
- [x] `pkg/protocol/header.go`: Current 74.25%,  Target **80%**, by @XiaoYi-byte 
- [x] `pkg/protocol/request.go`: Current 51.31%,  Target **80%**, Raised to 85.3%, by @BaiZe1998 
- [x] `pkg/protocol/response.go`: Current 60.14%(Overall 56%),  Target **80%**,  Raised to 81.8% by @gityh2021 
- [x] `pkg/protocol/multipart.go`: Current 20.25%,  Target **80%**, Raised to 79.75%,  by @gityh2021
- [x] `pkg/network`: Current 68.14%,  Target **80%**, Raised to 95.2%,    by @a631807682
- [ ] `pkg/common/utils`: Current  72.34%,  Target **80%**,  by 
- [x] `pkg/common/config`: Current 32.91%,  Target **80%**, Raised to 100%, by @rogerogers 
- [x] `pkg/common/adaptor`: Current 76.60%,  Target **85%**, Raised to 91.5%, by @L2ncE
- [x] `internal/bytesconv`: Current 1.68%,  Target **80%**, Raised to 82.35%, by @Alexdener
- [x]  `pkg/common/test`: Current 3.2%,  Target **80%**,  Raised to 98.1% by @a631807682 

Standard coverage stats: https://app.codecov.io/gh/cloudwego/hertz

### PR DESCRIPTION
**Pull request by sjcsjc123**
#### What type of PR is this?
test

#### Check the PR title.

- [x] This PR title match the format: `<type>(optional scope): <description>`.
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [x] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io).

#### (Optional) Translate the PR title into Chinese.
为`pkg/common/utils`下的`chunk_test.go`和`ioutil_test.go`添加一些场景测试

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en: Supplement unit test for `pkg/common/utils`, coverage increased from 72.34% to 88.1%
zh(optional): 补充 `pkg/common/utils` 的单元测试，覆盖率从 72.34% 提高到了 88.1%。

#### (Optional) Which issue(s) this PR fixes:
Fixes #257

### CODE REVIEW

File: pkg/common/utils/ioutil_test.go [hunk 1]
```diff
@@ -18,12 +18,86 @@ package utils
...
+	}
+}
+
+func newTestReaderForm(r io.ReaderFrom) readerTest {
+	return &testReader{
+		r: r,
+	}
+}
+
+func newLimitReader(r *bytes.Buffer) io.LimitedReader {
```
Comment (Duslia) [inline pkg/common/utils/ioutil_test.go:94]: Can it be reused or move it to mock dir so that other developers can use it in the future

Comment (sjcsjc123) [reply pkg/common/utils/ioutil_test.go:94]: of course, i have move it to mock dir

File: pkg/common/utils/chunk_test.go [hunk 2]
```diff
@@ -35,6 +35,33 @@ func TestChunkParseChunkSizeGetCorrect(t *testing.T) {
 	}
 }
 
+func TestChunkParseChunkSizeGetError(t *testing.T) {
+	// test err from -----n, err := bytesconv.ReadHexInt(r)-----
+	chunkSizeBody := ""
+	zr := mock.NewZeroCopyReader(chunkSizeBody)
+	chunkSize, err := ParseChunkSize(zr)
+	assert.DeepEqual(t, true, err != nil)
```
Comment (welkeyever) [inline pkg/common/utils/chunk_test.go:43]: better use "assert.NotNil()" instead of this

Comment (welkeyever) [reply pkg/common/utils/chunk_test.go:43]: the same goes for the following

Comment (sjcsjc123) [reply pkg/common/utils/chunk_test.go]: i have already modify them

**Review by Duslia — APPROVED** [review]

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

**Pull request merged**